dependencies/pom.xml

```diff
@@ -784,6 +784,16 @@
                 <artifactId>microstream-cache</artifactId>
                 <version>${version.lib.microstream}</version>
             </dependency>
+            <dependency>
+                <groupId>javax.cache</groupId>
+                <artifactId>cache-api</artifactId>
+                <version>1.1.1</version>
```

Comment:
You should not need to explicitly include this dependency. It is a transitive dependency of `microstream-cache`.  I was able to build your PR without expressing this dependency. So please remove it from here and from `integrations/microstream/cache/pom.xml`